consensus/test_definitions/defs.c

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2018 IOTA Stiftung
+ * https://github.com/iotaledger/entangled
+ *
+ * Refer to the LICENSE file for licensing information
+ */
+
+#include <string.h>
+
+#include "consensus/tangle/tangle.h"
+#include "consensus/test_definitions/defs.h"
+#include "utils/files.h"
+
+#define PATH_LENGTH 256
+
+static char *test_db = "consensus/cw_rating_calculator/tests/test.db";
```

Comment:
should be more generic path: `consensus/tests/test.db  and consensus/tests/test.db` ?

Reply:
You are reviewing an old commit, this has been removed due to the issue I told you (bazel virtualenv)